internal/zinc-core/src/main/scala/sbt/internal/inc/UsedName.scala

```diff
@@ -17,6 +17,6 @@ object UsedName {
   def apply(name: String, scopes: Iterable[UseScope] = Nil): UsedName = {
     val useScopes = util.EnumSet.noneOf(classOf[UseScope])
     scopes.foreach(useScopes.add)
-    UsedName(name, useScopes)
+    UsedName(name.replaceAllLiterally("\n", "\u2668"), useScopes)
```

Comment:
I think this should be "\u2668" + "0A" so if we come across another control code we can extend the solution.